engines/python/setup/djl_python/properties_manager/lmi_dist_rb_properties.py

```diff
@@ -82,6 +86,14 @@ def validate_speculative_and_lora(self):
             )
         return self
 
+    @model_validator(mode='after')
+    def validate_speculative_and_fml(self):
+        if self.load_format == LmiDistLoadFormats.sagemaker_fast_model_loader.value and self.speculative_draft_model:
+            raise AssertionError(
+                f"Cannot enable sagemaker_fast_model_loader and speculative decoding at the same time"
```

Comment:
What is the customer facing name for fast_model_loader? We should probably use the recognizable name here so customers know exactly what the incompatibility is. 

Once we have that, I would suggest changing this to `speculative decoding and <fast model loading name> are currently not compatible. Disable speculative decoding, or use standard checkpoints for the target model`